.github/workflows/lighthouse.yml

```diff
@@ -7,12 +7,16 @@ on:
     branches: [dev, staging, production]
 
 jobs:
-  check-diff:
```

Comment:
Was there a need to move this back to a stop instead of a separate job?

Reply:
Yes, it didn't work :)